include/rocksdb/table.h

```diff
@@ -246,29 +246,38 @@ struct BlockBasedTableOptions {
   // Default: 0 (disabled)
   uint32_t read_amp_bytes_per_bit = 0;
 
-  // We currently have five versions:
-  // 0 -- This version is currently written out by all RocksDB's versions by
-  // default.  Can be read by really old RocksDB's. Doesn't support changing
+  // format_version controls some features of newly written tables that
```

Comment:
A good inline comment would be succinct as otherwise nobody would read it. Most of the lines added could be inferred with a little thinking. Among the change I do find the following line useful though:
"format_version specifies how the newly generated SST files are encoded. The encoding type must be stored within SST file so that the code remains backward compatible."

Reply:
I didn't read the actual comment so I can't comment on whether it needs to be more concise. Just to clarify, this comment is not inline comment for source code, but comment to public interface. The reader will be users of RocksDB, not developers of it.